python/lsst/scarlet/lite/observation.py

```diff
@@ -326,7 +326,7 @@ def convolve(self, image: Image, mode: str | None = None, grad: bool = False) ->
                     newshape[2] += kernel.image.shape[2] - image.shape[2]
                 _image = _pad(_image, newshape)
                 result = convolve(_image, kernel.image, self.convolution_bounds)
-                result = centered(_image, image.data.shape)
+                result = centered(result, image.data.shape)  # type: ignore
```

Comment:
What is the `# type: ignore` about?

Reply:
It's because `convolve` returns either an `ndarray` or `Fourier` class. I could cast it to an `ndarray` but I find `typing.cast` to be clunky and I find it more readable to just ignore those types of mypy errors when I know what the code is doing.